src/Adapter/MSTest.CoreAdapter/Discovery/TypeEnumerator.cs

```diff
@@ -192,11 +192,8 @@ internal UnitTestElement GetTestFromMethod(MethodInfo method, bool isDeclaredInT
                 testElement.Description = descriptionAttribute.Description;
             }
 
-            var workItemAttributeArray = this.reflectHelper.GetCustomAttributes(method, typeof(WorkItemAttribute)) as WorkItemAttribute[];
-            if (workItemAttributeArray != null)
-            {
-                testElement.WorkItemIds = workItemAttributeArray.Select(x => x.Id.ToString()).ToArray();
-            }
+            var workItemAttributeArray = this.reflectHelper.GetCustomAttributes(method, typeof(WorkItemAttribute)).Cast<WorkItemAttribute>();
```

Comment:
Nit-picking: This is no longer an array. Please rename the variable.

Reply:
Not nit-picking, the variable name _is_ wrong

Removed `Attribute` Suffix